src/Calculator/Views/Settings.xaml

```diff
@@ -106,138 +79,103 @@
                       Margin="0,60,0,0"
                       Padding="24,0,24,16"
                       Style="{StaticResource SettingsContentScrollViewStyle}">
-            <Grid x:Name="ContentGrid">
+            <Grid>
                 <Grid.ColumnDefinitions>
-                    <ColumnDefinition x:Name="ColumnMain" Width="*"/>
-                    <ColumnDefinition x:Name="ColumnAbout" Width="0"/>
+                    <ColumnDefinition MaxWidth="1000"/>
```

Comment:
Is it necessary to keep this Grid if there is only one column left now?

Reply:
Ah good catch, removed it. Also added a RepositionTransition inline with other apps, so that cards smoothly animate whenever other cards expand.